### ISSUE TITLE
Cannot read GTF: expected 2 values got 1

### ISSUE THREAD

**Issue by almart7** [biocore-ntnu/pyranges#292]
Hello, I am trying to read the following GTF file: ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/vertebrate_mammalian/Mus_musculus/reference/GCF_000001635.27_GRCm39/GCF_000001635.27_GRCm39_genomic.gtf.gz
And got this error:

` Traceback (most recent call last):
  File "/home/alessandra/Desktop/Lab/isoAnnot/isoAnnot_dev/exons/layer_exons.py", line 85, in <module>
    main()
  File "/home/alessandra/Desktop/Lab/isoAnnot/isoAnnot_dev/exons/layer_exons.py", line 48, in main
    exons_info = get_gtf_exon_locations(args.gtf_file)
  File "/home/alessandra/Desktop/Lab/isoAnnot/isoAnnot_dev/exons/layer_exons.py", line 30, in get_gtf_exon_locations
    gtf_contents = pyranges.read_gtf(gtf_file, as_df=True, duplicate_attr=False)
  File "/home/alessandra/miniconda3/envs/ranges/lib/python3.10/site-packages/pyranges/readers.py", line 311, in read_gtf
    gr = read_gtf_full(f, as_df, nrows, _skiprows, duplicate_attr)
  File "/home/alessandra/miniconda3/envs/ranges/lib/python3.10/site-packages/pyranges/readers.py", line 343, in read_gtf_full
    extra = _to_rows(df.Attribute)
  File "/home/alessandra/miniconda3/envs/ranges/lib/python3.10/site-packages/pyranges/readers.py", line 367, in to_rows
    rowdicts.append({k: v
  File "/home/alessandra/miniconda3/envs/ranges/lib/python3.10/site-packages/pyranges/readers.py", line 368, in <dictcomp>
    for k, v in [kv.replace('""', '"NA"').replace('"', '').split(None, 1)
ValueError: not enough values to unpack (expected 2, got 1)
`

**Comment by endrebak** [comment]
It is probably a nonstandard file. I’ll have a look

**Comment by endrebak** [comment]
The file has `;` within double quotes which makes it hard to parse quickly.

**Comment by endrebak** [comment]
I propose implementing a correct but slow GTF reader that can be used if the normal one fail. There is no way to read all gtfs quickly since the format allows for such a degree of heterogeneity.
